http4k-security-oauth/src/main/kotlin/org/http4k/security/oauth/server/accesstoken/AuthorizationCodeAccessTokenGenerator.kt

```diff
@@ -78,7 +78,7 @@ private object AuthorizationCodeAccessTokenForm {
         with(accessTokenForm(request)) {
             AuthorizationCodeAccessTokenRequest(
                 clientId(this),
-                clientSecret(this),
+                clientSecret(this) ?: "",
```

Comment:
doesn't this want to be a nullable now?

Reply:
Maybe, I followed the pattern from places like this

ClientSecretAccessTokenRequestAuthentication

Though I think .orEmpty will be better so will do that